deploy/dev.yml

```diff
@@ -0,0 +1,107 @@
+version: "3"
+services:
+    dotcom:
+        build:
+            context: ../
+            dockerfile: ./deploy/dotcom/dev/Dockerfile
+        env_file:
+            - ../.env
+        depends_on:
+            - redis-cluster-init
+        environment:
+            - MIX_ENV=dev
```

Comment:
Since these values are elsewhere in this file anyway, I think it'd make sense to just add `REDIS_HOST=10.0.0.11` and `REDIS_PORT=6379` here

Reply:
The only issue there is that they're still needed for non-Docker development. So, it would split the environment variables based on your method of development. That was the idea with keeping them all in one file is that you only have to change them in one place no matter which dev method you use.

Reply:
Oh, also, the monitor needs those REDIS variables to run the healthcheck.

Reply:
Sounds good, thanks for clarifying!